codegen-client/src/test/kotlin/software/amazon/smithy/rust/codegen/client/smithy/generators/client/FluentClientGeneratorTest.kt

```diff
@@ -76,4 +88,50 @@ class FluentClientGeneratorTest {
             test = test,
         )
     }
+
+    @Test
+    fun `generate inner builders`() {
+        val test: (ClientCodegenContext, RustCrate) -> Unit = { codegenContext, rustCrate ->
+            rustCrate.integrationTest("inner_builder") {
+                val moduleName = codegenContext.moduleUseName()
+                rustTemplate(
+                    """
+                    ##[test]
+                    fn test() {
+                        let connector = #{TestConnection}::<#{SdkBody}>::new(Vec::new());
+                        let config = $moduleName::Config::builder()
+                            .endpoint_resolver("http://localhost:1234")
+                            #{set_http_connector}
+                            .build();
+                        let smithy_client = aws_smithy_client::Builder::new()
+                            .connector(connector.clone())
+                            .middleware_fn(|r| r)
+                            .build_dyn();
+                        let client = $moduleName::Client::with_config(smithy_client, config); 
+                    
+                        let say_hello_input_builder = client.say_hello().byte_value(4).foo("hello!");
```

Comment:
Nit: should probably be `fluent_builder` instead of `input_builder`

Reply:
Done.